assets/js/trivyScans.js

```diff
@@ -0,0 +1,335 @@
+      name: 'total_fixed_high',
+      createdCell: function (td, _cellData, rowData) {
+        const totalFixedHigh = rowData?.total_fixed_high || 0
+        $(td).html(totalFixedHigh)
+      },
+    },
+    {
+      data: 'total_fixed_medium',
+      name: 'total_fixed_medium',
+      visible: false,
+      createdCell: function (td, _cellData, rowData) {
+        const totalFixedMedium = rowData?.total_fixed_medium || 0
+        $(td).html(totalFixedMedium)
+      },
+    },
+    {
+      data: 'total_fixed_low',
+      name: 'total_fixed_low',
+      visible: false,
+      createdCell: function (td, _cellData, rowData) {
+        const totalFixedLow = rowData?.total_fixed_low || 0
+        $(td).html(totalFixedLow)
+      },
+    },
+    {
+      data: 'total_fixed_unknown',
+      name: 'total_fixed_unknown',
+      visible: false,
+      createdCell: function (td, _cellData, rowData) {
+        const totalFixedUnknown = rowData?.total_fixed_unknown || 0
+        $(td).html(totalFixedUnknown)
+      },
+    },
+    {
+      data: 'total_unfixed_critical',
+      name: 'total_unfixed_critical',
+      createdCell: function (td, _cellData, rowData) {
+        const totalUnfixedCritical = rowData?.total_unfixed_critical || 0
+        $(td).html(totalUnfixedCritical)
+      },
+    },
+    {
+      data: 'total_unfixed_high',
+      name: 'total_unfixed_high',
+      createdCell: function (td, _cellData, rowData) {
+        const totalUnfixedHigh = rowData?.total_unfixed_high || 0
+        $(td).html(totalUnfixedHigh)
+      },
+    },
+    {
+      data: 'total_unfixed_medium',
+      name: 'total_unfixed_medium',
+      visible: false,
+      createdCell: function (td, _cellData, rowData) {
+        const totalUnfixedMedium = rowData?.total_unfixed_medium || 0
+        $(td).html(totalUnfixedMedium)
+      },
+    },
+    {
+      data: 'total_unfixed_low',
+      name: 'total_unfixed_low',
+      visible: false,
+      createdCell: function (td, _cellData, rowData) {
+        const totalUnfixedLow = rowData?.total_unfixed_low || 0
+        $(td).html(totalUnfixedLow)
+      },
+    },
+    {
+      data: 'total_unfixed_unknown',
+      name: 'total_unfixed_unknown',
+      visible: false,
+      createdCell: function (td, _cellData, rowData) {
+        const totalUnfixedUnknown = rowData?.total_unfixed_unknown || 0
+        $(td).html(totalUnfixedUnknown)
+      },
+    },
+    {
+      data: 'total_secret_issues',
+      name: 'total_secret_issues',
+      visible: false,
+      createdCell: function (td, _cellData, rowData) {
+        const totalSecretIssues = rowData?.total_secret_issues || 0
+        $(td).html(totalSecretIssues)
+      },
+    },
+    {
+      data: 'result_link',
+      name: 'result_link',
+      createdCell: function (td, _cellData, rowData) {
+        const componentName = cleanColumnOutput(rowData.name || 'N/A')
+        $(td).html(rowData.result_link)
+      },
+    },
+  ]
+
+  let table
```

Comment:
Not sure this is needed?

Reply:
Or at least name it differently as `table` is used in quite a few places in functions, just so it's not confusing. Also for clarity move globals to the top of the file so they're easier to spot 🙂